blueprints/app/files/package.json

```diff
@@ -77,6 +77,7 @@
     "ember-qunit": "^8.1.0",
     "ember-resolver": "^13.0.2",
     "ember-source": "~6.0.0-beta.1",
+    "ember-template-imports": "^4.1.3",
```

Comment:
@NullVoxPopuli shouldn't we actually bring `@glint/environment-ember-template-imports` as we already install `@glint/environment-ember-loose` for ts apps?

and then I guess `tsconfig.json` should be updated with

```json
"environment": [
      "ember-loose",
      "ember-template-imports"
    ]
```

Reply:
yes!